elyra/elyra_app.py

```diff
@@ -97,6 +98,8 @@ def initialize_handlers(self):
             (f'/{self.name}/pipeline/components/{processor_regex}', PipelineComponentHandler),
             (f'/{self.name}/pipeline/components/{processor_regex}/{component_regex}/properties',
              PipelineComponentPropertiesHandler),
+            (f'/{self.name}/pipeline/components/{processor_regex}/{component_regex}/definition',
```

Comment:
Since the component definition is a strict subset of the properties API, I wonder if we need the `/definition` in the endpoint?  The thought being that `/pipeline/{processor_regex}/{component_regex}` gets you the definition and `/pipeline/{processor_regex}/{component_regex}/properties` gets you the definition in terms of the canvas properties - like its just a little more specific.

Reply:
Yeah I was toying with this idea too, especially since the definition handler class is basically a complete repeat of the properties handler class code